PocketKit/Sources/SaveToPocketKit/SavedItem/SavedItemViewModel.swift

```diff
@@ -143,7 +143,7 @@ extension SavedItemViewModel {
 
             if provider.hasItemConformingToTypeIdentifier(plainTextUTI) {
                 guard let string = try? await provider.loadItem(forTypeIdentifier: plainTextUTI, options: nil) as? String,
-                      let url = URL(string: string) else {
```

Comment:
We were receiving `"The Continentals first trailer takes the John Wick series to the 70s https://www.theverge.com/23680378/the-continental-teaser-trailer-john-wick-peacock"` as the string and so this value returned nil